src/plugin.js

```diff
@@ -228,70 +228,22 @@ const onPlayerReady = (player, emeError) => {
 
   setupSessions(player);
 
-  if (window.MediaKeys) {
+  const playerOptions = getOptions(player);
+  const isLegacyFairplay = playerOptions.keySystem && playerOptions.keySystem[LEGACY_FAIRPLAY_KEY_SYSTEM];
```

Comment:
Can we add some documentation regarding legacy/current fairplay for users?

Reply:
Yeah good call, will add some doc changes to this PR.

Reply:
Added some usage docs to the readme and a comment defining legacy fairplay.